src/main/kotlin/com/seokjoo/todo/domain/service/todo/TodoService.kt

```diff
@@ -97,4 +99,6 @@ class TodoService(
             }
         }
     }
+
+    private fun checkIsMe(todoOwnerId: String, userId: String) = todoOwnerId != userId
```

Comment:
```suggestion
    private fun isMe(todoOwnerId: String, userId: String) = todoOwnerId != userId
```

Boolean 리턴이니까 `check`는 없어도 충분해 보이네요 !

Reply:
좋아요 ! [96bd729](https://github.com/Ahn-seokjoo/todo-spring/pull/64/commits/96bd7299bce60a2acd032c783ef06f78ce388870)